xdcs-frontend/src/app/element/log-preview/log-preview.component.ts

```diff
@@ -35,4 +36,5 @@ export interface LogLine {
   lineNumber: number;
   time: Date;
   contents: string;
+  agentTag: String;
```

Comment:
tag, nie agentTag, po to jest użyte słowo tag żeby nie używać słowa agent, to jest inny poziom abstrakcji, który nie musi wiedzieć że to coś jest związane z agentem